qr-code-auth/README.md

```diff
@@ -0,0 +1,52 @@
+# QrCodeAuth
+
+## Requests
+
+1. POST 
+
+`/qr-code-auth?user-id=${user-id}&device-info={deviceInfo}`
+
+Description: Authenticates the user by verifying the user ID and device information.
+
+#### Request Body:
+
+```
+{
+  "user_id": "<user-id>",
+  "device_info": "<device-info>"
+}
+```
+
+#### Responses:
+
+```
+{
+    "statusCode": 201,
+    "userDeviceInfoData: {
+        "user_id": "9",
+        "device_info": "window"
+    },
+    "message": "User Device Info added successfully!"
+}
+```
+
+```
+{
+    "statusCode": 404,
+    "message": "Api not found."
+}
+```
+
+```
+{
+    "statusCode": 500,
+    "message": "Internal server error."
+}
+```
+
+
+
+
+
+
+
```

Comment:
These extra lines can be removed from the end.

Reply:
done